### ISSUE TITLE
Implement the baseline glove_solution.py

### ISSUE THREAD

**Issue by sergeivolodin** [EPFLMachineLearningTeamYoor/Project02#1]
The lab on text classification will be only after a few weeks. Therefore, it would be good to

1. See the lecture from the previous year from Martin
2. Find the lab task
3. Implement the lab task
4. Apply the knowledge obtained to write the `glove_template.py` file

**Comment by sergeivolodin** [comment]
Glove paper: https://nlp.stanford.edu/pubs/glove.pdf
Short description: https://nlp.stanford.edu/projects/glove/

**Comment by sergeivolodin** [comment]
Can reuse code from ex10 (movie recommendations)

**Comment by sergeivolodin** [comment]
Already done on lab, solution provided by TAs
